src/plugin/WindowsKeyboardCodes.h

```diff
@@ -0,0 +1,301 @@
+#ifndef VK_UNKNOWN
```

Comment:
File name does not conform with the style of other file names.

Reply:
yep, that's my bad, I realized it only after the pull request